cni/api/api.go

```diff
@@ -9,23 +9,19 @@ import (
 )
 
 type PodNetworkInterfaceInfo struct {
-	PodName        string
-	PodNamespace   string
+	PodName       string
+	PodNamespace  string
 	PodEndpointId string
-	ContainerID    string
-	IPAddresses    []net.IPNet
+	ContainerID   string
+	IPAddresses   []net.IPNet
 }
 
-type CNIState interface {
-	PrintResult() error
-}
-
-type AzureCNIState struct {
+type State struct {
 	ContainerInterfaces map[string]PodNetworkInterfaceInfo
 }
 
-func (a *AzureCNIState) PrintResult() error {
-	b, err := json.MarshalIndent(a, "", "    ")
+func (s *State) WriteToStdout() error {
```

Comment:
There are CNI changes in this PR. Does that mean you will cut CNI release before CNS?

Reply:
the CNI changes are superficial changes to the go client and do not need a CNI release or new binary
/cc @matmerr

Reply:
yeah I don't really have any opinions here, but potentially add a comment in addition to the rename mentioning `PrintResult` or `Print*`, as [CNI writes results to stdout via `PrintResult(result, version)` or more directly `result.Print()`](https://github.com/containernetworking/cni/blob/master/pkg/types/types.go#L103), and given the nonstandard cmd we're adding I'd like to have some trail to regex with when tracking down where CNI is returning a response